src/Sulu/Bundle/ResourceBundle/Resource/Exception/FilterContextNotFoundException.php

```diff
@@ -27,10 +27,10 @@ class FilterContextNotFoundException extends FilterException
      */
     private $name;
 
-    public function __construct($id)
+    public function __construct($name)
     {
         $this->entityName = 'SuluResourceBundle:Filter';
-        $this->id = $id;
+        $this->id = $name;
```

Comment:
you should rename property `id` to `name` if you rename the constructor argument!